src/main/java/ch/njol/skript/util/Date.java

```diff
@@ -67,18 +65,34 @@ public String toString() {
 	}
 	
 	/**
+	 * Get the timestamp of this date
+	 *
 	 * @return The timestamp in milliseconds
 	 */
 	public long getTimestamp() {
 		return timestamp;
 	}
 	
-	public void add(final Timespan span) {
+	/**
+	 * Add a {@link Timespan} to this date
+	 *
+	 * @param span Timespan to add
+	 * @return This date with the added timespan
+	 */
+	public Date add(final Timespan span) {
 		timestamp += span.getMilliSeconds();
+		return this;
 	}
 	
-	public void subtract(final Timespan span) {
+	/**
+	 * Subtract a {@link Timespan} from this date
+	 *
+	 * @param span Timespan to subtract
+	 * @return This dat with the subtracted timespan

```

Comment:
misspelled date

Reply:
Its the Canadian spelling? jk, Ill fix that!